### PR TITLE
Document Django Email in readme

### PR DESCRIPTION
**Pull request by OmarIthawi**
It was added in `0.1.7` via #25.

### CODE REVIEW

**Comment by OmarIthawi** [comment]
@mulby @ziafazal edX Bot didn't recognize this as an OSPR. Could you please take a look?

File: README.rst [hunk 1]
```diff
@@ -33,7 +34,8 @@ Even though they were written with ``edx-platform`` in mind, the guidelines
 should be followed for Open edX code in general.
 
 PR description template should be automatically applied if you are sending PR from github interface; otherwise you
-can find it it at `PULL_REQUEST_TEMPLATE.md <https://github.com/edx/edx-ace/blob/master/.github/PULL_REQUEST_TEMPLATE.md>`_
+can find it it
```
Comment (nasthagiri) [inline README.rst]: it it -> it

**Review by nasthagiri — APPROVED** [review]

**Comment by OmarIthawi** [comment]
Thanks @nasthagiri! Done.

**Comment by openedx-webhooks** [comment]
Although this pull request is already merged, I've created [OSPR-5539](https://openedx.atlassian.net/browse/OSPR-5539) so that we can track it in Jira.

There is nothing you have to do. No action is needed from your side.  Thanks again for your contribution.

**Pull request merged**